API/endpoints/CloudRecording.js

```diff
@@ -75,9 +73,8 @@ CloudRecording.listMeetingRecordings.scopes = [
  *   returned from a single API call
  * @param {string} [options.nextPageToken] - token used to pageinate
  *   through large result sets
- * @param {string} [options.searchTrashFor] - Indicate type of
- *   cloud recording to retreive from the trash.
- *   options - {'meeting_recordings', 'recording_file'}
+ * @param {string} [options.searchTrash=false] - set to true to retrieve
```

Comment:
Add documentation about the Zoom call behavior (i.e. that it uses default search trash recording type)?